go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/ugorji/go/codec v1.1.7
 	golang.org/x/net v0.0.0-20170828231752-66aacef3dd8a
 	gopkg.in/abiosoft/ishell.v1 v1.0.0-20171224170712-50251d04cb42
```

Comment:
This line can be removed

Reply:
can -> should! :-P

Reply:
Ah, also the first line is `github.com/abiosoft/ishell v2.0.0+incompatible // indirect`, which should be removed as well.